js/apps/system/_admin/aardvark/APP/frontend/js/views/graphViewer.js

```diff
@@ -1063,12 +1064,11 @@
 
       var callback = function (error, id, msg) {
         if (!error) {
-          var edge = {
-            source: from,
-            target: to,
-            id: id,
-            color: self.graphConfig.edgeColor || self.ecolor
-          };
+          var edge = _.clone(body);
```

Comment:
Might we loose our body variable scope here (callback)? I think it might be safer to do `this.editor.get()` as in Row: 1055.